server/project.clj

```diff
@@ -6,6 +6,7 @@
   :dependencies [[org.clojure/clojure "1.9.0-alpha14"]
                  [org.clojure/core.async "0.3.442"]
                  [org.clojure/data.json "0.2.6"]
+                 [org.clojure/java.jdbc "0.7.0-alpha3"]
```

Comment:
Don't think you need this b/c it's included as a dependency of the other db dependencies